markets/perps-market/contracts/storage/PerpsMarket.sol

```diff
@@ -238,17 +239,20 @@ library PerpsMarket {
         self.skew += newPositionSize - oldPositionSize;
 
         uint currentPrice = newPosition.latestInteractionPrice;
-        (int totalPositionPnl, , , , ) = oldPosition.getPnl(currentPrice);
+        (, int pricePnl, , int fundingPnl, , ) = oldPosition.getPnl(currentPrice);
 
         int sizeDelta = newPositionSize - oldPositionSize;
         int fundingDelta = calculateNextFunding(self, currentPrice).mulDecimal(sizeDelta);
         int notionalDelta = currentPrice.toInt().mulDecimal(sizeDelta);
 
         // update the market debt correction accumulator before losing oldPosition details
         // by adding the new updated notional (old - new size) plus old position pnl
-        self.debtCorrectionAccumulator += fundingDelta + notionalDelta + totalPositionPnl;
+        self.debtCorrectionAccumulator += fundingDelta + notionalDelta + pricePnl + fundingPnl;
```

Comment:
Should interestPaid not be included in the debtCorrectionAcc?

Reply:
this debt correction accumulator is per market whereas the interest rate for us is at the global level across all markets since the utilization rate is based on the supermarket's utilization.

the unrealized interest rate doesn't need to be reported since it's not "debt".  it's credit and when the credit is realized (when trades close) it gets deposited for LPs to gain.  think of it as a fee essentially when closing your trade except that since its continuous, it eats up your margin.

Reply:
That does make sense!

Reply:
Hmm although shouldn't that credit be reported as profit/negative debt and give the market more liquidity?

Reply:
No need to include unrealised interest in credit capacity